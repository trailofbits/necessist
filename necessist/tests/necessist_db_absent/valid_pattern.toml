args = ["--no-sqlite", "--root=examples/valid_pattern"]
stderr = ""

[bin]
name = "necessist"

[fs]
cwd = "../../.."

[status]
code = 0
