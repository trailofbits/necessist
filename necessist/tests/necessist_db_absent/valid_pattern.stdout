20 candidates in 1 test file
examples/valid_pattern/src/lib.rs: dry running
examples/valid_pattern/src/lib.rs: mutilating
examples/valid_pattern/src/lib.rs:49:5-49:20: `foo().method();` passed
examples/valid_pattern/src/lib.rs:49:10-49:19: `.method()` passed
examples/valid_pattern/src/lib.rs:50:5-50:26: `foo().method().baz();` passed
examples/valid_pattern/src/lib.rs:50:19-50:25: `.baz()` passed
examples/valid_pattern/src/lib.rs:51:5-51:28: `foo().ignored_method();` passed
examples/valid_pattern/src/lib.rs:52:5-52:34: `foo().ignored_method().baz();` passed
examples/valid_pattern/src/lib.rs:52:27-52:33: `.baz()` passed
examples/valid_pattern/src/lib.rs:54:5-54:18: `bar.method();` passed
examples/valid_pattern/src/lib.rs:55:5-55:24: `bar.method().baz();` passed
examples/valid_pattern/src/lib.rs:55:17-55:23: `.baz()` passed
examples/valid_pattern/src/lib.rs:57:5-57:32: `bar.ignored_method().baz();` passed
examples/valid_pattern/src/lib.rs:57:25-57:31: `.baz()` passed
examples/valid_pattern/src/lib.rs:59:5-59:24: `bar.field.method();` passed
examples/valid_pattern/src/lib.rs:60:5-60:30: `bar.field.method().baz();` passed
examples/valid_pattern/src/lib.rs:60:23-60:29: `.baz()` passed
